plugins/user_management_plugin.py

```diff
@@ -62,40 +86,73 @@ class UserManagementModelView(ModelView):
     #     email=dict(validators=[])
     # )
 
+    def on_form_prefill(self, form, id):
+        if (not is_super_user(self)):
+            form.username.render_kw = {'readonly': True}
+            form.superuser.render_kw = {'readonly': True}
+
     def on_model_change(self, form, model, is_created):
         logging.info("UserManagementModelView.on_model_change(form=" + str(form) + ", model=" + str(model) + ", is_created=" + str(is_created) + ")")
+        logging.info("Confirm Password:" + form.password_confirm.data )
```

Comment:
We should remove this so the password is not logged

Reply:
Removed the password from logging.